### PR TITLE
Replaced "the open source browser from Google"

### PR DESCRIPTION
**Pull request by maytees**
Replaced "the open source browser from Google" and replaced it with "the web browser from Google", because Google Chrome is not open source.

### CODE REVIEW

**Comment by Whobeu** [comment]
It is true that "Chrome" is not open source but "Chromium" is.  Perhaps "Chrome" should be replaced with "Chromium" which would negate having to change the rest of the text.

**Comment by jakobkummerow** [comment]
This repository is a read-only mirror, so we can't accept any PRs here.

Logistics aside, the factual merit of this PR is at the very least debatable:

> [In short, Google Chrome is the Chromium open source project built, packaged, and distributed by Google.](https://chromium.googlesource.com/chromium/src/+/main/docs/chromium_browser_vs_google_chrome.md)

IMHO calling Google Chrome an "open source browser" is a good summary of that situation, so nitpicking on the wording isn't a good use of anyone's time.

**Comment by KazimirPodolski** [comment]
@jakobkummerow The page you gave _somehow_ forgets to mention Google's blobs and spyware.

**Comment by jakobkummerow** [comment]
@KazimirPodolski While I am aware that some folks on the Internet like to claim otherwise, the truth is that the Google Chrome team is extremely privacy conscious. For details, please read the [Chrome Privacy Whitepaper](https://www.google.com/chrome/privacy/whitepaper.html).

TL;DR: There is no spyware in Chrome; anyone claiming otherwise is spreading FUD.

That said, the V8 GitHub mirror still isn't the place to discuss Chrome (nor other browsers).
